e2e/cypress/integration/channel/user_to_channel_admin_member_updates_manage_channel_members_modal_spec.js

```diff
@@ -71,7 +71,7 @@ describe('Change Roles', () => {
         });
     });
 
-    it('MM-10858 - Going from a Channel Member to Channel Admin update view member modal without refresh', () => {
+    it('MM-T4174 User role to channel admin/member updates channel member modal immediately without refresh', () => {
```

Comment:
Updated test ID only. This fails on daily due to https://mattermost.atlassian.net/browse/MM-38104